src/Hl7.Fhir.Core.Tests/ElementModel/PocoTypedElementTests.cs

```diff
@@ -21,7 +21,7 @@ public class PocoTypedElementTests
         public void PocoTypedElementPocoRoundtrip()
         {
             var patient = new Patient();
-            var actual = patient.ToTypedElement().ToPoco<Patient>();
+            var actual = patient.ToTypedElement().ToPoco<Patient>(ModelInfo.ModelInspector);
```

Comment:
Isn't this ToPoco<T> a shim that can do this `ModelInfo.ModelInspector` itself?  I think you showed me that yesterday.

Reply:
Solved